src/main.rs

```diff
@@ -159,6 +170,48 @@ mod tests {
         assert_eq!(output.stdout.matches("\n").count(), 8, "stdout = {:?}", output.stdout);
     }
 
+    #[test]
+    fn it_defaults_to_hex_output() {
+        let output = get_success(&["./src/tests/red.png", "--max-colours=1", "--no-palette"]);
+
+        assert_eq!(output.exit_code, 0);
+
+        assert!(
+            output.stdout == "#ff0000\n" || output.stdout == "#fe0000\n",
```

Comment:
nit: possibly this is `cargo fmt` disagreeing with me, in which case I'll live with it, but the line break after the `||` operator was intentional – it aligns the two outputs vertically, so they're easier to compare.